docs/source/guides/integrations.mdx

```diff
@@ -0,0 +1,170 @@
+# Integrate any ML framework with the Hub
+
+The Huggingface Hub makes it easy to host and share models with the community. It already has support for
+[dozens of libraries](https://huggingface.co/docs/hub/models-libraries) in the Open Source ecosystem. We are always
+working on expanding this support to push Machine Learning forward. The `huggingface_hub` library plays a key role in
+this process, allowing any Python script to easily push and load files.
+
+When talking about integrating a library to the Hub, we have 4 main topics:
+1. **Push to Hub:** implement a `push_to_hub()`-like method to upload a model to the Hub.
+   This includes the model weights as well as the modelcard and any other relevant information/data necessary to run the model (example: logs from training).
```

Comment:
```suggestion
   This includes the model weights, as well as [the model card](https://huggingface.co/docs/huggingface_hub/how-to-model-cards) and any other relevant information or data necessary to run the model (for example, training logs).
```
This is completely optional but I thought it kinda assumes too much that the person knows all about the model card (and I feel like it's the second most important asset after weights)